srcpkgs/luanti/template

```diff
@@ -9,10 +9,20 @@ makedepends="MesaLib-devel freetype-devel gmp-devel libcurl-devel
  libjpeg-turbo-devel libopenal-devel libvorbis-devel lua52-devel
  sqlite-devel libXi-devel"
 depends="desktop-file-utils hicolor-icon-theme"
-short_desc="InfiniMiner/Minecraft inspired game"
+short_desc="Voxel game-creation platform with easy modding and game creation"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://www.minetest.net"
 changelog="https://dev.minetest.net/Changelog"
 distfiles="https://github.com/minetest/minetest/archive/${version}.tar.gz"
-checksum=aa9a6ae57445b779f57dcba5a83b0704fabd24c5eca37c6c8611e885bdf09d7c
+checksum=2a3161c04e7389608006f01280eda30507f8bacfa1d6b64c2af1b820a62d2677
+
+post_install() {
+	vinstall misc/net.minetest.minetest.desktop 644 usr/share/applications
```

Comment:
did this change name?

Reply:
if it did and there's still a compat desktop file, maybe that can go in the minetest package

Reply:
It did not, I didn't dig into the build files to see why it didn't get included like it did with previous builds.